whois-api/src/test/java/net/ripe/db/whois/api/ApiKeyAuthServerDummy.java

```diff
@@ -57,6 +60,9 @@ public class ApiKeyAuthServerDummy implements Stub {
         APIKEY_TO_OAUTHSESSION.put(BASIC_AUTH_PERSON_OWNER_MNT, new OAuthSession(AUD, "l6lRZgvOFIphjiGwtCGuLwqw","person@net.net", "906635c2-0405-429a-800b-0602bd716124", "profile email whois.mntner:OWNER-MNT"));
         APIKEY_TO_OAUTHSESSION.put(BASIC_AUTH_TEST_TEST_MNT, new OAuthSession(AUD, "hHZjAbXPtxGxUJCgdwv2ufhY","test@ripe.net", "8ffe29be-89ef-41c8-ba7f-0e1553a623e5", "whois.mntner:TEST-MNT profile email"));
         APIKEY_TO_OAUTHSESSION.put(BASIC_AUTH_INVALID_SIGNATURE_API_KEY, new OAuthSession(AUD, "hHZjAbXPtxGxUJCgdwv2ufhY","invalid@ripe.net", "8ffe29be-89ef-41c8-ba7f-0e1553a623e5", "profile email whois.mntner:TEST-MNT"));
+        APIKEY_TO_OAUTHSESSION.put(BASIC_AUTH_FULL_ENV_SIGNATURE_API_KEY, new OAuthSession(AUD, "3TE0KYX35SZQUKIABM5VAOGW","test@ripe.net", "906635c2-0405-429a-800b-0602bd716124", "whois.environment:LOCALHOST.mntner:OWNER-MNT.write"));
+        APIKEY_TO_OAUTHSESSION.put(BASIC_AUTH_NOT_MNT_ENV_SIGNATURE_API_KEY, new OAuthSession(AUD, "SJ9691NXUKZFVZMQXLQYBVRH","test@ripe.net", "906635c2-0405-429a-800b-0602bd716124", "whois.environment:LOCALHOST.write"));
+        APIKEY_TO_OAUTHSESSION.put(BASIC_AUTH_FULL_WRONG_ENV_SIGNATURE_API_KEY, new OAuthSession(AUD, "RQMO5ASNRUN8ZJNP6TCS1Q2Q","test@ripe.net", "906635c2-0405-429a-800b-0602bd716124", "whois.environment:PREPDEV.write"));
```

Comment:
Please add the profile and email scope as well in the list.

Reply:
Done